### PR TITLE
Include missing imports

### PR DESCRIPTION
**Pull request by rndnoise**
Maybe these are obvious, but these imports need to be added for the program to compile. I'm enjoying your tutorial and using the library, thanks for writing it. I'm looking forward to reading the next section on execution contexts!

### CODE REVIEW

File: docs/http_server.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ main = HTTPure.serve 8080 router $ Console.log "Running server..."
 The body of the request that our server accepts has to be in JSON format. Now that we have the body of the request we can try and parse it. The body _must_ contain the `query` variable and additionally may also contain `variables` and `operationName`. We will write a function `decodeParams` that takes a JSON value and returns a record containing the important GraphQL request parameters.
 
 ```purescript
+import Data.Maybe
```
Comment (hendrikniemann) [inline docs/http_server.md]:
```suggestion
import Data.Maybe (Maybe)
```
Maybe just import the needed type?

**Review by hendrikniemann — COMMENTED** [review]
Do you think adding the imports is helpful? I felt like the imports are rather distracting from the actual code. I don't really know what the tutorial audience is. Is it PureScript programmers trying out GraphQL or is it GraphQL/JavaScript experts trying out PureScript? I guess adding the imports does not hurt to much and will help PureScript beginners. Could you do the little proposed change? Thanks for contributing and the kind words. ❤️

**Comment by hendrikniemann** [comment]
Sorry for the weird references, I accidently tagged this PR instead of the issue. It's way to late for coding 😄

**Comment by rndnoise** [comment]
I'm not sure about how useful the imports are... ones like `Data.Either` and `Data.Maybe` are probably more noisy than helpful. Though I had to search for some like the functions in Argonaut. I think you'll get a better sense of who is using the library over time (people new to PureScript or new to GraphQL, or neither). Hopefully for now, maybe being verbose will help someone :-)

**Comment by hendrikniemann** [comment]
I merged this in 1d24a7e

**Pull request closed without merging**